Code/Framework/AzToolsFramework/AzToolsFramework/Prefab/Instance/Instance.h

```diff
@@ -37,6 +37,7 @@ namespace AzToolsFramework
 
         using AliasPath = AZ::IO::Path;
         using AliasPathView = AZ::IO::PathView;
+        using RootAliasPath = AliasPath;
```

Comment:
nit: I recommend moving this to PrefabFocusHandler so that it has more context to what it corresponds to. I don't think the core Instance API itself needs to worry too much about the root instance/alias. That's the job of either the EOS or the public handlers.

Reply:
I was hoping to start using RootAliasPath as an Instance Id/handler of sorts throughout the UI side, since Prefabs lack something like that and I don't want to expose Instance Refs for multiple reasons. 

Potentially it could be moved to the public handler, but then I would not want to expose the function to convert RootAliasPath to InstanceOptionalReference, so it would still need to reside somewhere lower.

Not sure if you already had something in mind for instance handler alternatives, so maybe we could discuss that and have a single system for everything?

Reply:
Would it be possible to add a comment/description for what exactly `RootAliasPath` is (as well as `AliasPath`), otherwise to the uninitiated it can be quite opaque (or maybe just a link to another doc/reference)

Reply:
I think RootAliasPath can reside in PrefabEditorEOS since that's where the root prefab instance lives. So, it makes more sense to put it there than the Instance class itself.

Reply:
Guess it makes sense.
If you don't mind, I would try to get this change in, and then submit a follow-up PR with both this change and the refactoring of PrefabFocusHandler to remove the hierarchy variable.